SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -7114,6 +7114,69 @@ void CEulerSolver::BC_Inlet(CGeometry *geometry, CSolver **solver_container,
           break;
       }
 
+      /*--- Check if the inlet node is shared with a viscous wall. ---*/
+
+      if (geometry->nodes->GetViscousBoundary(iPoint)) {
+
+        switch (Kind_Inlet) {
+
+          /*--- Total properties have been specified at the inlet. ---*/
+
+          case INLET_TYPE::TOTAL_CONDITIONS: {
+
+            /*--- Impose the wall velocity from the interior wall node. ---*/
+
+            Velocity2 = 0.0;
+            for (iDim = 0; iDim < nDim; iDim++) {
+              V_inlet[iDim+1] = nodes->GetVelocity(iPoint,iDim);
+              Velocity2 += V_inlet[iDim+1] * V_inlet[iDim+1];
+            }
+
+            /*--- Match the pressure, density and energy at the wall. ---*/
+
+            Pressure = nodes->GetPressure(iPoint);
+            Density = Pressure / (Gas_Constant * Temperature);
```

Comment:
This temperature comes from the normal stagnation treatment?
Total conditions already respects the velocity of the interior node, what is this changing?
And, can we instead change the PTotal and TTotal that are specified and let the normal treatment do the rest?